docs/proposals/mvp-dashboard-design.md

```diff
@@ -43,18 +43,82 @@ can query the relevant test builds.
 
 ## Proposal
 
-![Diagram 1](../images/perf-dash-proposal-diagram.png)
+![alt text](../images/perf-dash-proposal-diagram.png "Perf-Dash Diagram")
+
+### Server API
+
+The system we propose requires the development of a server API that will be
+used to serve queries that a user makes via the dashboard. To speed up the
+initial development process, these queries will be hardcoded for test results.
+Adding query build functionality for the user is a feature that will be
+developed in future development cycles.
+
+The server polls the data storage unit for performance test logs. The frequency
```

Comment:
- 'unit' dropped
- changed the text to clarify that the server is querying on demand.  My bad, I misunderstood how the server communicated with storage.  See the new commits for updated text.